pkgs/by-name/fl/flow-editor/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  zig,
```

Given the volatile nature of Zig, I think it'd be better if we pin it to a specific version
```suggestion
  zig_0_13,
```